From: Adam Saponara <as@php.net>
Date: Fri, 28 Dec 2018 03:20:51 -0500
Subject: fix submodule init in `vendor/Makefile`

---
 vendor/Makefile | 17 +++++++++--------
 1 file changed, 9 insertions(+), 8 deletions(-)

diff --git a/vendor/Makefile b/vendor/Makefile
index 9617dee..cb0b114 100644
--- a/vendor/Makefile
+++ b/vendor/Makefile
@@ -3,18 +3,18 @@ lua_objects:=$(patsubst lua/%.c,lua/%.o,$(filter-out lua/lua.c, $(wildcard lua/*
 termbox_cflags:=-std=c99 -Wall -Wextra -pedantic -Wno-unused-result -g -O3 -D_XOPEN_SOURCE $(CFLAGS)
 termbox_objects:=$(patsubst termbox/src/%.c,termbox/src/%.o,$(wildcard termbox/src/*.c))
 
-all: ./pcre/.libs/libpcre.a ./lua/liblua5.3.a ./termbox/src/libtermbox.a vendor/uthash/.git
+all: uthash/.git pcre/.libs/libpcre.a lua/liblua5.3.a termbox/src/libtermbox.a
 
-vendor/uthash/.git:
+uthash/.git:
 	git submodule update --init
 
-./pcre/.libs/libpcre.a:
+pcre/.libs/libpcre.a:
 	cd pcre && ./autogen.sh && ./configure && $(MAKE)
 
-./lua/liblua5.3.a: $(lua_objects)
+lua/liblua5.3.a: $(lua_objects)
 	$(AR) rcs $@ $(lua_objects)
 
-./termbox/src/libtermbox.a: $(termbox_objects)
+termbox/src/libtermbox.a: $(termbox_objects)
 	$(AR) rcs $@ $(termbox_objects)
 
 $(lua_objects): %.o: %.c
@@ -24,8 +24,9 @@ $(termbox_objects): %.o: %.c
 	$(CC) -c $(termbox_cflags) $< -o $@
 
 clean:
-	rm -f ./lua/liblua.a $(lua_objects)
-	rm -f ./termbox/src/libtermbox.a $(termbox_objects)
-	$(MAKE) -C pcre clean
+	rm -f lua/liblua5.3.a $(lua_objects)
+	rm -f termbox/src/libtermbox.a $(termbox_objects)
+	rm -f pcre/config.h.in~
+	-$(MAKE) -C pcre clean
 
 .PHONY: all clean
